projects/dino/config/dino_vit-base-p16_8xb64-amp-coslr-100e_in1k.py

```diff
@@ -0,0 +1,103 @@
+model = dict(
+    type='DINO',
+    data_preprocessor=dict(
+        mean=[123.675, 116.28, 103.53],
+        std=[58.395, 57.12, 57.375],
+        bgr_to_rgb=True),
+    backbone=dict(type='mmcls.VisionTransformer', arch='b', patch_size=16),
```

Comment:
```suggestion
    backbone=dict(type='mmpretrain.VisionTransformer', arch='b', patch_size=16),
```
please use model in mmpretrain

Reply:
done